src/CMakeLists.txt

```diff
@@ -17,19 +16,9 @@ configure_file (
     "${PROJECT_SOURCE_DIR}/include/objCrypto/version.h"
 )
 
-# controll static shared with BUILD_SHARED_LIBS=ON.
-add_library(
-  objCrypto
-)
-
-target_sources( objCrypto 
-  PRIVATE
-  objCrypto.cc
-  aes-ctr.cc 
-  aes-gcm.cc )
+add_library( objCrypto  aes-ctr.cc aes-ctr.h aes-gcm.cc aes-gcm.h objCrypto.cc )
```

Comment:
I'm worried that this loses the PRIVATE on the sources

Reply:
Fixed.